### DISCUSSION TITLE
Resume training possible?

### DISCUSSION THREAD

**Discussion by Eleksar387** [d8ahazard/sd_dreambooth_extension#148] [Q&A]
Working good for me on a 3060/12GB! Thx!!!

Is it possible to resume training if I have the feeling I need to run it more? What settings do I need to change then?

**Comment by Jonseed** [comment]
I think you just reload the model, load params from the model, and click train. It will start where it left off last time.

**Comment by d8ahazard** [comment]
What he said.
